### ISSUE TITLE
Not able to change scopes

### ISSUE THREAD

**Issue by dreenot** [CodetrixStudio/CapacitorGoogleAuth#97]
Hello, my app has two different stages of authentication.
It is an Ionic tabs ionic-angular capacitor project.

The first with scopes for read only and the second with scopes for taking actions in the user's account.
This may be the case of other developers too, who doesn't want to ask for all the permissions at once.

The problem is that all the configuration for this plugin are passed in **capacitor.config.json**

I've spent the last days trying to figure out a way to workaround it, with no results.
It would be a great improvement if you could pass the config as a parameter in **GoogleAuth.signIn()**

If you know some way to do it, please let me know, even if it is not elegant.

**Comment by DexterHuang** [comment]
yes, please, I don't need to have user's profile, and it could drive away many privacy conscious users

**Comment by DilipTharuka** [comment]
same issue, any workaround?
